src/gui/CMakeLists.txt

```diff
@@ -204,8 +204,9 @@ target_link_libraries(owncloud owncloudCore)
 
 find_package(Qt5LinguistTools)
 if(Qt5LinguistTools_FOUND)
-    qt5_add_translation(client_I18N ${TRANSLATIONS})
-    target_sources(owncloud PRIVATE ${client_I18N})
+    qt5_add_translation(client_compiled_translations ${TRANSLATIONS})
+    target_sources(owncloud PRIVATE ${client_compiled_translations})
+    add_resources_to_target(owncloud translations.qrc translations "${client_compiled_translations}")
```

Comment:
a qrc should be part of the source files

Reply:
That's done by `target_sources` in the `add_resources_to_target` function The icons function also uses `target_sources`. Instead of "returning" the name of the generated file, I thought it's easier to avoid return parameters, but instead just pass the target name and let the function figure out the rest.